src/components/interfaces/MOBILE_API.xml

```diff
@@ -797,6 +797,13 @@
     <element name="DYNAMIC" />
   </enum>
 
+  <enum name="DeliveryMode">

```

Comment:
@anosach-luxoft  As I know if we change HMI API we should increase it's minor version version.